packages/@ama-sdk/core/package.json

```diff
@@ -101,6 +101,7 @@
     "@nx/eslint-plugin": "~17.1.1",
     "@nx/jest": "~17.1.1",
     "@o3r/build-helpers": "workspace:^",
+    "@o3r/core": "workspace:^",
```

Comment:
I think it should be also in `peer optional` to give hint to the user of the version of @o3r/core used in the definition